packages/functional_widget/pubspec.yaml

```diff
@@ -15,6 +15,9 @@ dependencies:
   source_gen: ^1.1.0
   code_builder: ^4.0.0
   collection: ^1.0.0
+    
+dependency_overrides:
```

Comment:
What is the purpose of this dependency override

Reply:
Will need to upgrade `mockito` to get rid of it.